src/proj.rs

```diff
@@ -584,6 +584,17 @@ impl Proj {
         transform_epsg(ctx, from, to, area)
     }
 
+    /// Returns epsg code for Proj
+    /// 
+    /// # Safety
+    /// This method contains unsafe code.
+    pub fn to_epsg(&self) -> Result<&str, str::Utf8Error> {
```

Comment:
I'd rather call it `identifier` or `code` or `id_code` or `authority_code`. Not all of them are EPSG.

Reply:
Thank you for your suggestion. I have revised the PR accordingly。